src/Microsoft.AspNetCore.Mvc.Abstractions/Properties/Resources.Designer.cs

```diff
@@ -15,304 +15,266 @@ internal static class Resources
         /// </summary>
         internal static string ArgumentCannotBeNullOrEmpty
         {
-            get { return GetString("ArgumentCannotBeNullOrEmpty"); }
```

Comment:
Just do this in a separate PR and #YOLO it. You know you want those sweet sweet #PranavPoints